tensorrt_llm/serve/disagg_auto_scaling.py

```
@@ -109,12 +117,38 @@ async def watch_workers(self, get_existing_first: bool = True):
+            logger.warning(
+                f"Initializing watch task with {len(workers)} existing workers")
+            for worker_info in workers:
+                await on_event(worker_info, WatchEventType.SET)
+            logger.warning("Start watching worker events")
+            while True:
+                try:
+                    worker_events = await self._watch_handle.drain()
+                    for event in worker_events:
+                        worker_info = self._parse_worker_info(event)
+                        await on_event(worker_info, event.event_type)
+                except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)